### DISCUSSION TITLE
White dots on model

### DISCUSSION THREAD

**Discussion by ukito-pl** [google/model-viewer#2540] [Q&A]
### Description

Hi, i recently bumped into an issue with random white dots on model:
![image](https://user-images.githubusercontent.com/19152578/124243889-a8a7c080-db1e-11eb-850c-cc08bb9876ea.png)
Unfortunately it happens only on one device so far. Any other models are working fine on this device except for this one and on any other device I have tested it also works fine and I have no clue what might be the reason behind it.

#### Live Demo
Here is the model where this behavior occurs:
https://app.seemymodel.com/models?k=0e5ab9616d9c9df483b07b3c

### Version

- model-viewer: v1.7.1

### Browser Affected
- [x] Chrome ver. 91.0.4472.114
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [x] Windows 10 Enterprise 1909 build 18363.1316

**Comment by elalish** [comment]
This looks like a GPU driver bug to me. These are very hard to fix or work around, so if it's only this one device, I don't think there's much we can do. You can try filing a bug on Windows.

**Reply by shamsail** [reply]
I have tried the link https://app.seemymodel.com/models?k=0e5ab9616d9c9df483b07b3c with windows 10 and chrome browser and it's giving fine results to me.
![aa](https://user-images.githubusercontent.com/30554980/124762251-2408d800-df4c-11eb-9266-491f23e0e06a.png)

what was the problem? machine/gpu? or model-viewer: v1.7.1? @ukito-pl

**Reply by ukito-pl** [reply]
Thank you for checking it out. Unfortunately I don't have access to the device it happened on and on any other device it is fine. If I have some more information I will post it here.
